js/components/FileChooser.tsx

```diff
@@ -131,14 +131,14 @@ const FilePickerChooser = ({
         } finally {
             setBusy(false);
         }
-    }, []);
+    }, [accept, busy, onChange]);
```

Comment:
This will just redefine the callback when `busy` changes, right? Seems superfluous, but I guess it doesn't hurt.

Reply:
Yeah, it errors on the side of caution. Most of the things it flagged I know to be invariant, but it doesn't hurt. And if that assumption changes, then you're covered.